packages/block-editor/src/components/link-control/index.js

```diff
@@ -404,20 +404,24 @@ function LinkControl( {
 								createSuggestionButtonText
 							}
 							hideLabelFromVision={ ! showTextControl }
+							// Passing the Button component as a suffix prop
+							suffix={
+								showActions ? undefined : (
+									<Button
```

Comment:
We want this button to be `size="small"`, and wrap it in a `<InputControlSuffixWrapper variant="control">`.

See this [With Icon Or Control story](https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-inputcontrol--docs#with-icon-or-control) for an example.

Reply:
> We want this button to be `size="small"`, and wrap it in a `<InputControlSuffixWrapper variant="control">`.
> 
> See this [With Icon Or Control story](https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-inputcontrol--docs#with-icon-or-control) for an example.

Sure, I would change this.